### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by shoom3301**

### CODE REVIEW

File: cypress-custom/integration/swap.test.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ describe('Swap (custom)', () => {
     cy.get('#swap-currency-input .token-amount-input').should('be.visible')
     cy.get('#swap-currency-input .token-amount-input').type('0.05', { force: true, delay: 400 })
     cy.get('#swap-currency-output .token-amount-input').should('not.equal', '')
+    cy.get('#fees-exceed-checkbox').then((el) => {
+      if (el) el.click()
```
Comment (alfetopito) [inline cypress-custom/integration/swap.test.ts:46]: What about

```suggestion
      el?.click()
```

Comment (alfetopito) [reply cypress-custom/integration/swap.test.ts:46]: Also, can't you instead of `.then()` do a `.click()`?

**Review by alfetopito — APPROVED** [review]
Nice fix!
Just some small questions

File: cypress-custom/integration/swap.test.ts (hunk 1 above)
Comment (shoom3301) [reply cypress-custom/integration/swap.test.ts:46]: Unfortunately I can't, because the `#fees-exceed-checkbox` element availability is optional

Comment (alfetopito) [reply cypress-custom/integration/swap.test.ts:46]: ~But, don't we control when it's visible or not?~
Oh, does this depend on the answer from the API? Then yeah, I see your point

**Pull request merged**